webhooks-extension/test/cleanup.sh

```diff
@@ -9,8 +9,11 @@ source ${test_dir}/util.sh
 if [ $DASHBOARD_INSTALL_NS != "default" ]; then
   kubectl delete ns ${DASHBOARD_INSTALL_NS}
 else
-  kubectl delete deployment tekton-dashboard -n $DASHBOARD_INSTALL_NS
+  kubectl delete deployment tekton-dashboard -n default
```

Comment:
`$DASHBOARD_INSTALL_NS` rather than default?

Reply:
No it's explicitly default because it's in the 'else if namespace==default' section.